test/blackbox-tests/test-cases/reporting-of-cycles.t/run.t

```diff
@@ -42,24 +42,22 @@ error message.
 
   $ echo 'val x : unit' > indirect/c.mli
   $ dune build @indirect-deps
-  File "indirect/dune", line 6, characters 0-43:
-  6 | (alias
-  7 |  (name indirect-deps)
-  8 |  (deps a.exe))
   Error: dependency cycle between modules in _build/default/indirect:
      A
   -> C
   -> A
+  -> required by indirect/a.exe
+  -> required by alias indirect/indirect-deps in indirect/dune:6
   [1]
 
 But when the cycle is due to the cmi files themselves, the message becomes
 cryptic and can involve unrelated files:
 
   $ echo 'val xx : B.t' >> indirect/c.mli
   $ dune build @indirect-deps
-  Error: Dependency cycle between the following files:
-     _build/default/indirect/.a.eobjs/a.impl.all-deps
-  -> _build/default/indirect/.a.eobjs/b.impl.all-deps
-  -> _build/default/indirect/.a.eobjs/c.intf.all-deps
-  -> _build/default/indirect/.a.eobjs/a.impl.all-deps
+  Error: Dependency cycle between:
+     indirect/.a.eobjs/a.impl.all-deps
```

Comment:
Do these paths actually make sense? They look like they should only exist under `_build`.

Reply:
Indeed. When printing paths, we generally drop the `_build/default` part. But technically it's only safe to do so for files that are copied from the source tree. It should be possible to do something about this.

Reply:
Alright, should be fixed here and more generally in all dep stacks.